### ISSUE TITLE
Doesn't work on Linux

### ISSUE THREAD

**Issue by ghost** [p2sr/Portal2SpeedrunMod#1]
The game just instantly crash. It works using Proton but at the cost of some visual bugs.

**Comment by ThisAMJ** [comment]
The solution is pinned in the #technical-help channel in [the P2SR Discord server](https://discord.com/invite/p2sr). I recommend you join it if you ever need tech support in the future.

> Edit Portal 2 Speedrun Mod/gameinfo.txt, and replace the line `Game portal2` with the absolute path to your Portal 2 install, like this:
```
SearchPaths
{
	Game                |gameinfo_path|.
	Game                "/home/<USERNAME>/.steam/steam/steamapps/common/Portal 2/portal2"
}
```

**Comment by mlugg** [comment]
For the record, this will also be fixed in the next P2SM version

**Comment by NekomusuMae** [comment]
> 



> The solution is pinned in the #technical-help channel in [the P2SR Discord server](https://discord.com/invite/p2sr). I recommend you join it if you ever need tech support in the future.
> 
> > Edit Portal 2 Speedrun Mod/gameinfo.txt, and replace the line `Game portal2` with the absolute path to your Portal 2 install, like this:
> 
> ```
> SearchPaths
> {
> 	Game                |gameinfo_path|.
> 	Game                "/home/<USERNAME>/.steam/steam/steamapps/common/Portal 2/portal2"
> }
> ```

Maybe it would be good to put that information in the readme instead of a pinned message on a channel in the mod's Discord server. Atleast until it's fixed.

**Comment by ThisAMJ** [comment]
I agree, but the server is for speedrunning all the games in the Portal 2 vein, not just Speedrun Mod. All of the top runners are active there.

**Comment by ThisAMJ** [comment]
VTC as completed

**Comment by mlugg** [comment]
give me literally 10 more minutes to release 1.6 and i'll close this

**Comment by mlugg** [comment]
Fixed in 1.6
